### PR TITLE
Complete the pass_gen.py file

### PR DESCRIPTION
**Pull request by mostafakavand**
pass_gen.py complete. For this, it was necessary to make small changes to other files. The code is executed as follows: (for example, we want to use pgpw)

`python pass_gen.py pgpw`

### CODE REVIEW

File: WORDS/PGPEW/pgpew_service.py [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
-                for j in range(len(prob_pass_list)):
-                    # building our string password by joining probable words in [j] index
-                    prob_pass_string = ''.join(prob_pass_list[j])
-                    # finally should write the password in specific file
-                    file.write(f"{prob_pass_string} \n")
-    except:
-        print("Something get fucked! Try again or try another service")
\ No newline at end of file
+def permutations_function(*, file_path, start_range, end_range, word_list):
+    with ExitStack() as stack:
```
Comment (BamdadTabari) [inline WORDS/PGPEW/pgpew_service.py:23]: please add comment for new things to describe your code

File: pass_gen.py [hunk 2]
```diff
@@ -1,13 +1,72 @@
...
+from WORDS.PGPW import pgpw
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description="Password Generator Tool")
+    parser.add_argument("method", type=str, help="Method to create a list of passwords (pgcw, pgcwrw, pgpew, or pgpw)")
+    args = parser.parse_args()
+
+    if args.method == 'pgcw':
```
Comment (BamdadTabari) [inline pass_gen.py:58]: it's good, we just need to add a help command to help the user!

**Review by BamdadTabari — APPROVED** [review]
job is done. see the issues part for new stuff

File: pass_gen.py (hunk 2 above)
Comment (mostafakavand) [reply pass_gen.py:58]: It exists.
The user can access it by typing `-h`

Comment (mostafakavand) [reply pass_gen.py:58]: `python pass_gen.py -h`

File: WORDS/PGPEW/pgpew_service.py (hunk 1 above)
Comment (mostafakavand) [reply WORDS/PGPEW/pgpew_service.py:23]: This item will be added as soon as possible

File: pass_gen.py (hunk 2 above)
Comment (BamdadTabari) [reply pass_gen.py:58]: needs more details

**Pull request merged**